ddtrace/propagation/http.py

```diff
@@ -678,6 +678,16 @@ def _extract(headers):
             meta=meta,
         )
 
+    @staticmethod
+    def _inject(span_context, headers):
+        # type: (Context, Dict[str, str]) -> None
+        tp = span_context._traceparent
```

Comment:
Question: can context be None?

Reply:
No, if a span is asked to return context and there is None, it will return `Context(trace_id=self.trace_id, span_id=self.span_id)`